### ISSUE TITLE
Build a traffic light classifier

### ISSUE THREAD

**Issue by ljanyst** [kung-fu-panda-automotive/carla-driver#9]
Take an image as input and figure out which light is on.

Check this out: http://cvrr.ucsd.edu/vivachallenge/index.php/traffic-light/traffic-light-detection/

**Comment by asimonov** [comment]
will implement in pure tensorflow, not keras

**Comment by asimonov** [comment]
trying to download data from http://cvrr.ucsd.edu/vivachallenge/index.php/traffic-light/traffic-light-detection/. They require academic email to register. I asked if I can use my gmail as I am udacity student.

**Comment by asimonov** [comment]
for now I am creating a simple CNN-based tensorflow classifier.

also looking at Bosch dataset: https://github.com/bosch-ros-pkg/bstld
it is called 'small' but it is actually a lot of data

**Comment by asimonov** [comment]
extracted data from Udacity ROS bags. getting black/white images.

here is the steps: https://github.com/kung-fu-panda-automotive/carla-driver/wiki/Extract-data-from-ROS-bags

asked on slack if something is wrong. we need RGB, right?

**Comment by asimonov** [comment]
made progress with conv net model to classify images on Bosch dataset. the activity is happening on https://github.com/asimonov/Bosch-TL-Dataset. 

state at the moment: model works. trains to good accuracy (98%).

TODO: 
1. finalise the internals to follow good practice of working with tensorflow models
2. put final class and trained weights under carla-driver
